Makefile

```diff
@@ -312,57 +312,77 @@ undeploy-c9s-%-c9s-python-3.9: bin/kubectl
 	$(info # Undeploying notebook from $(NOTEBOOK_DIR) directory...)
 	$(KUBECTL_BIN) delete -k $(NOTEBOOK_DIR)
 
+# Function for testing readiness by pinging the /api endpoint
+#   ARG 1: Image tag name.
+define test_notebook
+	$(eval FULL_NOTEBOOK_NAME := $(shell $(KUBECTL_BIN) get pods -l app=$(1) -o custom-columns=":metadata.name" | tr -d '\n'))
```

Comment:
I don't think this one particluar line shoud be part of this function, I would prefer to keep it out of here.

Reply:
Why not?

Reply:
seems like the FULL_NOTEBOOK_NAME is not being used in test_notebook.
probably moving it to the line 351 , might work